template/content/clients.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+var TChannel = require('tchannel');
+var HyperbahnClient = require('tchannel/hyperbahn');
+var fs = require('fs');
+var path = require('path');
+var Ringpop = require('ringpop');
+var Logtron = require('logtron');
+var Statsd = require('uber-statsd-client');
+var os = require('os');
+var process = require('process');
+var myLocalIp = require('my-local-ip');
+var Uncaught = require('uncaught-exception');
+var ProcessReporter = require('process-reporter');
+
+var thriftFile = fs.readFileSync(
+    path.join(__dirname, 'thrift', 'service.thrift'), 'utf8'
+);
+
+/* TODO:
+    - heap dump
+    - repl
+*/
+module.exports = ApplicationClients;
+
+function ApplicationClients(config, options) {
+    /*eslint no-process-env: 0*/
```

Comment:
I'd rather you deleted this and put `var NODE_ENV =  require('process').env.NODE_ENV;` at the top.